app/renderer/abr-document.js

```diff
@@ -789,6 +813,18 @@ AbrDocument.prototype = {
         return spellchecker.isMisspelled;
     },
 
+    // Adds the given word to our custom spellcheck-ok words
+    addSpelledWord: function (word) {
```

Comment:
To avoid duplicates in config (in case of multiple windows), maybe we can add this here:

```javascript
this.words = this.getConfig("spellchecker:custom-words");
```